pages/bookmark-page.js

```diff
@@ -105,18 +113,28 @@ const StyledWineGlasButton = styled.button`
 
 const StyledBarrelButton = styled.button`
   font-family: Georgia, 'Times New Roman', Times, serif;
-  font-size: 45px;
   color: #8a98a5;
   border: none;
-  margin-top: 5px;
+  margin-top: 3px;
   background-color: rgba(31, 31, 35, 1);
   :hover {
     border-bottom: 3px solid white;
     padding-top: 3px;
   }
 `;
+
+const WineFitsTo = styled.p`
+  font-weight: 500;
+  margin-top: 0.5rem;
+`;
+
+const WineType = styled.p`
+  font-weight: 500;
+  margin-top: 0.5rem;
+`;
+
```

Comment:
the upper styled components are identical. does it maybe make sense to just have one styled component instead?